typedoc.js

```diff
@@ -1,19 +1,16 @@
 module.exports = {
   name: 'Polkadot JS API',
-  mode: 'file',
-  exclude: ['test', '**/*spec.ts'],
+  exclude: [],
   excludeExternals: true,
   excludeNotExported: true,
   excludePrivate: true,
   hideGenerator: true,
   includes: './docs/guides',
   includeDeclarations: true,
-  out: 'docs/api',
+  out: 'docs',
   module: 'commonjs',
-  stripInternal: 'true',
-  theme: 'markdown',
-  ignoreCompilerErrors: true,
   moduleResolution: 'node',
-  suppressExcessPropertyErrors: true,
-  suppressImplicitAnyIndexErrors: true
-}
+  mdEngine: 'gitbook',
+  stripInternal: 'true',
+  theme: 'markdown'
```

Comment:
In package.json, you pass `--theme default --out docs/html`, which overwrites options here.

Reply:
Ahh, package.json is actually not used. (But I didn't want to remove it ... yet, it helps when you can actually test HTML from typedoc as well) 

This is a whole sprawling thing, effectively what is actually called is the build-docs script, which is shared - https://github.com/polkadot-js/dev/blob/master/packages/dev/scripts/polkadot-dev-build-docs.sh

(Hence build calling build-ts & build-docs)

Reply:
Ah, right, this is the obscure part I missed.

Reply:
Obscure is the right word, yes.